forge/test/operators/pytorch/matmul/test_matmul.py

```diff
@@ -0,0 +1,295 @@
+    (((1, 13, 89, 3),                     None, None, None),  pytest.mark.xfail(reason=FailingReasons.UNSUPPORTED_DIMENSION)),   # RuntimeError: TT_THROW @...
+    (((3, 11, 45, 17),                    None, None, None),  pytest.mark.xfail(reason=FailingReasons.COMPILATION_FAILED)),      # RuntimeError: Unhandled attribute type
+    (((2, 2, 3, 4),                       None, None, None),  pytest.mark.xfail(reason=FailingReasons.COMPILATION_FAILED)),      # RuntimeError: Unhandled attribute type
+    (((4, 11, 1, 23),                     None, None, None),  pytest.mark.xfail(reason=FailingReasons.COMPILATION_FAILED)),      # RuntimeError: Unhandled attribute type
+    (((5, 11, 64, 1),                     None, None, None),  pytest.mark.xfail(reason=FailingReasons.COMPILATION_FAILED)),      # RuntimeError: Unhandled attribute type
+    (((6, 100, 100, 100),                 None, None, None),  pytest.mark.xfail(reason=FailingReasons.BUGGY_SHAPE)),             # AssertionError
+    (((7, 10, 1000, 100),                 None, None, None),  pytest.mark.xfail(reason=FailingReasons.COMPILATION_FAILED)),      # RuntimeError: Unhandled attribute type
+    (((8, 1, 10, 1000),                   None, None, None),  pytest.mark.xfail(reason=FailingReasons.COMPILATION_FAILED)),      # RuntimeError: Unhandled attribute type
+    (((11, 32, 32, 64),                   None, None, None),  pytest.mark.xfail(reason=FailingReasons.BUGGY_SHAPE)),             # AssertionError
+    (((12, 64, 160, 96),                  None, None, None),  pytest.mark.xfail(reason=FailingReasons.BUGGY_SHAPE)),             # AssertionError
+    (((13, 11, 17, 41),                   None, None, None),  pytest.mark.xfail(reason=FailingReasons.COMPILATION_FAILED)),      # RuntimeError: Unhandled attribute type
+    (((14, 13, 89, 3),                    None, None, None),  pytest.mark.xfail(reason=FailingReasons.BUGGY_SHAPE)),             # AssertionError
+
+    (((1, 3, 3), None, forge.DataFormat.Int8,      forge.MathFidelity.HiFi4),  pytest.mark.xfail(reason=FailingReasons.BUGGY_SHAPE)),   # AssertionError
+    (((1, 3, 3), None, forge.DataFormat.RawUInt8,  forge.MathFidelity.HiFi4),  pytest.mark.xfail(reason=FailingReasons.BUGGY_SHAPE)),   # AssertionError
+    (((1, 3, 3), None, forge.DataFormat.RawUInt32, forge.MathFidelity.HiFi4),  pytest.mark.xfail(reason=FailingReasons.BUGGY_SHAPE)),   # AssertionError
+    (((1, 3, 3), None, forge.DataFormat.RawUInt16, forge.MathFidelity.HiFi4),  pytest.mark.xfail(reason=FailingReasons.BUGGY_SHAPE)),   # AssertionError
+    (((1, 3, 3), None, forge.DataFormat.UInt16,    forge.MathFidelity.HiFi4),  pytest.mark.xfail(reason=FailingReasons.BUGGY_SHAPE)),   # AssertionError
+)
+# fmt: on
+
+
+# Example of how to run just specific case(s) - first skip all, then enable what case to run:
+utils.extend_shape_params_with_marks(
```

Comment:
Do we want to comment this one out? does this call reset marks for all shapes?

Reply:
This is an example of how can easily skip all other tests and execute just specified.
I'll comment all, thanks!